CRISPResso2/CRISPRessoReports/CRISPRessoReport.py

```diff
@@ -86,22 +94,31 @@ def add_fig_if_exists(fig_name, fig_root, fig_title, fig_caption, fig_data,
                 html_string += html.read()
                 html_string += "</div>"
             amplicon_figures['htmls'][fig_name] = html_string
+        elif os.path.exists(jsonfullpath) and pro_installed:
+            root_name = fig_root.replace('.', '_').replace('-', '_')
+            d3_nuc_quilt_names.append(f"nuc_quilt_{root_name}")
+            with open(jsonfullpath) as fig_json_fh:
+                amplicon_figures['htmls'][fig_name] = f"""
+                <div class="d-flex justify-content-between" style="max-height: 80vh; overflow-y: auto;" id="{f"nuc_quilt_{root_name}"}"></div>
+                <script type="text/javascript">const {f"nuc_quilt_{root_name}"} = {fig_json_fh.read().strip()}</script>
+                    """
 
 
 def assemble_figs(run_data, crispresso_folder):
     """
         Helper function create the data structre for the figures
     """
     figures = {'names': {}, 'locs': {}, 'titles': {}, 'captions': {}, 'datas': {}, 'htmls': {}, 'sgRNA_based_names': {}}
+    d3_nuc_quilt_names = []
```

Comment:
d3_nuc_quilt_names needs to be available to pass as data. So I think we need to remove the default from the function stub so it always needs to be passed.

Reply:
It still looks like `d3_nuc_quilt_names` will always be `[]`, which would indicate that no data is being passed. I could definitely be missing something, but either we aren't properly passing `d3_nuc_quilt_names` or it can be defaulted.

Reply:
I think I understand now... `d3_nuc_quilt_names` is passed through and is added to each time in the `add_fig_if_exists` function and then it is eventually returned from `assemble_figs`.

Let's leave it as it is (and hopefully clean up how all of this plotting is handled later)!